### PR TITLE
More for Michaels comments

### LINKED ISSUE #46: Michaels review - "unspecified termination"
Regarding this part:

> A From Packet Number without an Until Packet Number has an unspecified termination.

His comment is:

> A From Packet Number without an Until Packet Number has an unspecified termination.

### PR DESCRIPTION
**Pull request by MaxF12**
Closes #46, #43, and #45

### CODE REVIEW

File: draft-jholland-quic-multicast.md [hunk 1]
```diff
@@ -197,6 +197,9 @@ Since clients can join later than a channel began, it is RECOMMENDED that client
 Clients should therefore begin with a high initial_max_streams_uni or send an early MAX_STREAMS type 0x13 value (see Section 19.11 of {{RFC9000}}) with a high limit.
 Clients MAY use the maximum 2^60 for this high initial limit, but the specific choice is implementation-dependent.
 
+The same stream ID may be used in both one or more multicast channels and the unicast connection.
```
Comment (GrumpyOldTroll) [inline draft-jholland-quic-multicast.md:200]: I think this paragraph needs to refer to https://www.rfc-editor.org/rfc/rfc9000#section-2.2, and basically needs to restate what it says there and emphasize that this spec changes nothing about that handling, it only means there are extra ways that a client endpoint might receive data for a stream multiple times.  I don't think we have a new MUST here, we just have the same MUST from RFC 9000 section 2.2.

**Comment by MaxF12** [comment]
Now also closes #24

File: draft-jholland-quic-multicast.md [hunk 2]
```diff
@@ -340,9 +344,9 @@ MC_CHANNEL_PROPERTIES frames contain the following fields:
...
   * ACK Bundle Size: The minimum number of ACKs a client should send in a single QUIC packet. If the max_ack_delay would force a client to send a packet that only consists of MC_CHANNEL_ACK frames, it SHOULD instead wait with sending until at least the specified number of acknowledgements have been collected. However, the Client MUST send any pending acknowledgements at least once per Max Idle Time to prevent the Server from perceiving the channel as interrupted.
 
-From Packet Number and Until Packet Number are used to indicate the packet number (Section 17.1 of {{RFC9000}}) the 1-RTT packets received over which these values are applicable.
+From Packet Number and Until Packet Number are used to indicate the packet numbers (Section 17.1 of {{RFC9000}}) of the 1-RTT packets received for which the values contained in a MC_CHANNELS_PROPERTIES frame are applicable.
 
-A From Packet Number without an Until Packet Number has an unspecified termination.
+If a frame contains a From Packet Number without an Until Packet Number, there is no specified termination to the applicability of its values.
```
Comment (GrumpyOldTroll) [inline draft-jholland-quic-multicast.md:349]: There is always now an Until Packet Number, but I think the value 0 could be used here to indicate indefinite termination.

**Review by GrumpyOldTroll — COMMENTED** [review]
Looks good with one nit

File: draft-jholland-quic-multicast.md [hunk 3]
```diff
@@ -230,14 +232,16 @@ Since clients can join later than a channel began, it is RECOMMENDED that client
 Clients should therefore begin with a high initial_max_streams_uni or send an early MAX_STREAMS type 0x13 value (see Section 19.11 of {{RFC9000}}) with a high limit.
 Clients MAY use the maximum 2^60 for this high initial limit, but the specific choice is implementation-dependent.
 
+The same stream ID may be used in both one or more multicast channels and the unicast connection. Streams behave exactly as specified in Section 2.2 of {{RFC9000}}. However, clients need to be able to handle cases where they receive the same stream data on multiple channels.
```
Comment (GrumpyOldTroll) [inline draft-jholland-quic-multicast.md:235]: Better, but there's still something here bothering me.

How about:

~~~
Stream frames for server-initiated streams can be sent on multicast
channels as well as the unicast connection.  Doing so is only possible
with stream data that's identical for the same offset and stream ID
across all receivers on that channel.   Stream data for the same offset
might also be sent on the unicast channel or on other multicast
channels.  As described in Section 2.2 of {{RFC9000}}, stream data
received multiple times for the same offset MUST be identical;
if it's not identical it MAY be treated as a connection error of type
PROTOCOL_VIOLATION.
~~~

Maybe that's too wordy, not sure.  The main issues I'm trying to make different in this text:
- I think it's wrong to say it's exactly as in section 2.2 of 9000 because we do differ on flow control, which I think we cover in the next section but also has a MUST in 2.2 of 9000.
- The "however" sentence seems strange because handling it when you get the same stream data is already required in 9000 (but is phrased as "received multiple times" instead of "received on multiple channels").

File: draft-jholland-quic-multicast.md [hunk 4]
```diff
@@ -233,14 +233,15 @@ Clients should therefore begin with a high initial_max_streams_uni or send an ea
...
 
 Instead, clients advertise resource limits that the server is responsible for staying within via MC_CLIENT_LIMITS ({{client-limits-frame}}) frames and their initial values from the transport parameter ({{transport-parameter}}).
 The server advertises the expected maxima of the values that can contribute toward client resource limits within a channel in MC_CHANNEL_PROPERTIES ({{channel-properties-frame}}) frames.
 
-If the server asks the client to join a channel that would exceed the client's limits with an up-to-date Client Limit Sequence Number, the client should send back a MC_CLIENT_CHANNEL_STATE_CHANGE ({{client-channel-state-frame}}) with "Declined Join" and reason "Property Violation".
+If the server asks the client to join a channel that would flexceed the client's limits with an up-to-date Client Limit Sequence Number, the client should send back a MC_CLIENT_CHANNEL_STATE_CHANGE ({{client-channel-state-frame}}) with "Declined Join" and reason "Property Violation".
```
Comment (GrumpyOldTroll) [inline draft-jholland-quic-multicast.md:244]: nit: typo "flexceed" -> "exceed"
